src/app/Dashboard/components/DashboardContainer.js

```diff
@@ -78,6 +92,7 @@ export class DashboardContainer extends Component {
     this.setState({ user: userInfo });
     fetchUserActivites(userInfo.id);
     loadCategories();
+    document.addEventListener('mousedown', this.hideFilter, false);
```

Comment:
I also think we should destroy or rather remove the event listener once the component `unmount` as it's no longer needed, leaving the listener might conflict with other components that has listeners on them.

So, we can do something like:
```
componentWillUnmount() {
  document.removeEventListener('mousedown', this.hideFilter, false);
}
```